lochness/redcap/__init__.py

```diff
@@ -107,6 +108,13 @@ def initialize_metadata(Lochness: 'Lochness object',
     df = pd.DataFrame()
     # for each record in pulled information, extract subject ID and source IDs
     for item in data:
+        if multistudy:
+            site_two_letters_redcap_id = item[redcap_id_colname][:2]
+            site_two_letters_study = study_name.split('_')[1]
+
+            if not site_two_letters_redcap_id == site_two_letters_study:
```

Comment:
Better constructed as `site_two_letters_redcap_id != site_two_letters_study`

Reply:
> Hi Kevin, I am back to this work. Please guide me a bit before I read your code:
> 
> > we are utilizing the first two letters from the subject ID to be matched to the last two letters of the study name given
> 
> Consider the following three IDs:
> 
> ```
> AD70163
> YA24873
> LA54257
> ```
> 
> To be able pull them multi-site folder hierarchy, should the sync command be:
> 
> > sync.py --source redcap -c config.yml --studies FAKE_AD GENUINE_YA UNKNOWN_LA

Yes. It's currently constructed to be used like how you've structured your `sync.py` command.

Reply:
> Better constructed as `site_two_letters_redcap_id != site_two_letters_study`

Thanks for catching this.
Updated here - https://github.com/PREDICT-DPACC/lochness/pull/32/commits/477ab26e8022083596c528f070381133dcf6240a

Reply:
Thank you.

> Yes. It's currently constructed to be used like how you've structured your sync.py command.

So I must create multiple study directories with empty metadata files and then execute:

> sync.py --source redcap -c config.yml --studies FAKE_AD GENUINE_YA UNKNOWN_LA ...

Right?

Reply:
>So I must create multiple study directories with empty metadata files and then execute:

Correct. Or you could also use `lochness_create_template.py` to create the PHOENIX structure for the studies for you.

Reply:
Also, please provide a `.lochness.json` for this purpose.